hibernate-core/src/main/java/org/hibernate/engine/spi/SessionFactoryDelegatingImpl.java

```diff
@@ -195,34 +193,6 @@ public IdentifierGenerator getIdentifierGenerator(String rootEntityName) {
 		return delegate.getProperties();
 	}
 
-	@Deprecated
-	public EntityPersister getEntityPersister(String entityName) throws MappingException {
-		return delegate.getRuntimeMetamodels()
-				.getMappingMetamodel()
-				.getEntityDescriptor( entityName );
-	}
-
-	@Deprecated
-	public Map<String, EntityPersister> getEntityPersisters() {
-		// for the time being, leave this calling `MetamodelImplementor` to avoid
-		// creating the map
-		return delegate.getMetamodel().entityPersisters();
-	}
-
-	@Deprecated
-	public CollectionPersister getCollectionPersister(String role) throws MappingException {
-		return delegate.getRuntimeMetamodels()
-				.getMappingMetamodel()
-				.getCollectionDescriptor( role );
-	}
-
-	@Deprecated
-	public Map<String, CollectionPersister> getCollectionPersisters() {
-		// for the time being, leave this calling `MetamodelImplementor` to avoid
-		// creating the map
-		return delegate.getMetamodel().collectionPersisters();
-	}
-
```

Comment:
@sebersole IMO, `SessionFactoryDelegatingImpl` doesn't need to declare new public methods :)

Reply:
(I mean "new" in the sense of not inherited from its supertypes.)

Reply:
OTOH, perhaps I'm wrong. Not sure. If they're needed, they need docs explaining why.